ICAFIX/hcp_fix_multi_run

```diff
@@ -112,7 +112,7 @@ opts_AddOptional '--training-file' 'TrainingData' 'somefile.RData, somefile.pyfi
 
 opts_AddOptional '--fix-threshold' 'FixThresh' 'number' 'Threshold setting in FIX that controls the sensitivity/specificity tradeoff, default = 10' '10'
 
-opts_AddOptional '--T1wTemplateBrain' 'T1wTemplateBrain' 'somefile.nii.gz' "path to T1w template brain, used by pyfix" "${HCPPIPEDIR}/global/templates/MNI152_T1_1mm_brain.nii.gz"
+opts_AddOptional '--T1wTemplateBrain' 'T1wTemplateBrain' 'somefile.nii.gz' "path to T1w template brain, used by pyfix for creating vein masks. Should have same resolution as subject structural data." "${HCPPIPEDIR}/global/templates/MNI152_T1_0.7mm_brain.nii.gz"
```

Comment:
Are we using 0.7 mm as the "default" in other pipelines (going back to the HCP-YA resolution)?
Because, at this point, given Lifespan and AABC, 0.8 mm might otherwise be considered a better default.

Reply:
I thought we decided to get rid of that option entirely.  It says Outdated above...

Reply:
Seems to currently exist, but have no default (empty string).

Reply:
Right it was the default we were getting rid of.